### PR TITLE
Add conversion of ConvOp from ONNX to TOSA

### PR DESCRIPTION
**Pull request by maxbartel**
This implements the conversion of `ConvOp` from ONNX  to TOSA. Since TOSA does not support the `group` attribute special handling is needed to support it.

Signed-off-by: Maximilian Bartel <maximilian.bartel@amd.com>

### CODE REVIEW

File: src/Conversion/ONNXToTOSA/DialectBuilder.cpp [hunk 1]
```diff
@@ -100,6 +100,19 @@ Value TosaBuilder::transpose(mlir::Value &value, llvm::ArrayRef<int32_t> perm) {
   return newValue;
 }
 
+Value TosaBuilder::slice(Value &inputConst, llvm::ArrayRef<int64_t> size,
```
Comment (AlexandreEichenberger) [inline src/Conversion/ONNXToTOSA/DialectBuilder.cpp:103]: thanks for hiding the handling of the type inside the builder, that is exactly what builders are for.

**Review by AlexandreEichenberger — APPROVED** [review]
LGTM

**Review by eric-k256 — COMMENTED** [review]
Nice solution for the grouped channels issue. Looks good from a TOSA perspective.

**Comment by maxbartel** [comment]
Thank you for the reviews! @AlexandreEichenberger Can you merge this please if it passes the pipeline?

**Pull request merged**